src/docs/markdown/command-line.md

```diff
@@ -495,6 +503,51 @@ The address of this request can be customized using the `--address` flag, or fro
 If you want to stop the current configuration but do not want to exit the process, use [`caddy reload`](#caddy-reload) with a blank config, or the [`DELETE /config/`](/docs/api#delete-configpath) endpoint.
 
 
+### `caddy storage`
+
+Allows export and import of the contents of Caddy's configured data storage.
+
+This is useful when needing to transition from one [storage module](/docs/json/storage/) to another, by exporting from your old one, updating your config, then importing into the new one.
+
+The following command can be used to do this all in one shot, using and old and new config with different configured storage modules, piping the export output into the 
```

Comment:
Into the  ...

Reply:
Crap :man_facepalming: I'm too scatterbrained sometimes and half finish sentences. I'll fix it.